src/main/kotlin/com/exactpro/th2/lwdataprovider/entities/requests/SseEventSearchRequest.kt

```diff
@@ -118,8 +125,23 @@ data class SseEventSearchRequest(
             throw InvalidRequestException("One of the 'startTimestamp' or 'resumeFromId' must not be null")
     }
 
-    fun checkRequest() {
+    private fun checkRequest() {
         checkStartPoint()
         checkEndTimestamp()
     }
+
+    private fun getInitEndTimestamp(passedEndTimestamp: Instant?) : Instant? {
+        return if (resultCountLimit != null && passedEndTimestamp == null){
+            if(searchDirection == TimeRelation.AFTER){
+                Instant.MAX
+            } else Instant.MIN
```

Comment:
Please, use brackets for all if statments
```suggestion
            if(searchDirection == TimeRelation.AFTER){
                Instant.MAX
            } else {
                Instant.MIN
            }
```